kindtest.sh

```diff
@@ -287,18 +287,27 @@ docker ps
 echo 'Clean up result root...'
 rm -rf "${RESULT_ROOT:?}/*"
 
+# If the IT_TEST is not set to "**/It*" (i.e. individual tests are chosen), 
```

Comment:
# If IT_TEST is set, integration-test profile is used to run the tests and 
# MAVEN_PROFILE_NAME parameter is ignored

Reply:
done